packages/scheduler/src/forks/SchedulerMock.js

```diff
@@ -414,6 +414,7 @@ function cancelHostTimeout(): void {
 
 function shouldYieldToHost(): boolean {
   if (
+    (expectedNumberOfYields === 0 && yieldedValues === null) ||
```

Comment:
What's this about?

Reply:
You can do

```
expect(Scheduler).toFlushAndYieldThrough([]);
```

Reply:
`yieldedValues === null` doesn't seem necessary